.github/workflows/cflite_pr.yml

```
@@ -0,0 +1,30 @@
+name: ClusterFuzzLite PR fuzzing
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ main ]
```

Currently the main branch is named as master, because it is a bit old library. I will change the name at v4, but it takes time. For now, use the current name.
```suggestion
    branches: [ master ]
```